R/LSgameofchance.R

```diff
@@ -15,10 +15,7 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 
-library(MGLM)
-library(HDInterval)
-library(DT)
-library(MCMCpack)
+
```

Comment:
We don't call `library` inside of JASP analysis. It should be handled similarly as in R packages, unless you are calling a function from the base R package (i.e., `matrix`), you always use `package::function`.